Objective-C/Tests/ReplicatorTest+Main.m

```diff
@@ -805,9 +805,6 @@ - (void) testDocumentReplicationEventAfterReplicatorStops {
     [self waitForExpectations: @[xc1] timeout: 5.0];
     [r removeChangeListenerWithToken: token1];
     
-    // CBL-4391 : Add a small delay to avoid database locked issue
```

Comment:
What is this?

Reply:
It is a workaround to make test passed when using lite-core/3.1-355 before we decided to make the fix for RC3.